test/e2e/nsx_networkinfo_test.go

```diff
@@ -318,3 +327,9 @@ func deleteVPCNetworkConfiguration(t *testing.T, ncName string) {
 	})
 	require.NoError(t, err)
 }
+
+func le(err error) {
```

Comment:
This function's name is unclear, and it seems unnecessary to wrap it.

Reply:
fixed